tests/XrdHttpTests/XrdHttpTests.cc

```diff
@@ -0,0 +1,167 @@
+#undef NDEBUG
```

Comment:
So, I am confused. Wasn't the test suite spun off into a separate pull request as @amadio requested? Am seeing something wrong here?

Reply:
I think it's not necessary to split into another pull request. Just a separate commit within this pull request is fine. I just want to cherry-pick just the fix for the patch release, and leave the tests just on master, since it depends on GoogleTest.